### DISCUSSION TITLE
HERO8 installation help

### DISCUSSION THREAD

**Discussion by Jax1414** [gopro/labs#390] [General]
Removes as unrelated

**Comment by Jax1414** [comment]
Downloaded Hero8 firmware update per instructions and "check" mark displayed on front and back screens. Downloaded QR Control app to iPhone. Created QR code for Motion on the app. 

With camera "on", pointed lens at QR code as displayed on app but camera did not acknowledge it. Repeated everything and same result.

Is it necessary to enable QR code recognition on a GoPro menu?

I would have thought the firmware from Labs  includes auto  recognition. Camera info in About states version 2.51.

Ideas? Suggestions?

Thank you.

**Comment by dnewman-gpsw** [comment]
That is not Labs firmware.  Labs version for that camera is 2.51.75.

**Comment by dnewman-gpsw** [comment]
Labs Firmware is only available here https://gopro.github.io/labs/

**Reply by Jax1414** [reply]
Thank you for the prompt response.
